internal/service.go

```diff
@@ -89,6 +91,21 @@ func moduleEnabledInEnv(identifier string) bool {
 	return false
 }
 
+// moduleEnabledInProductFiles returns true if the provided `identifier` is
+// a name of a file in the  /etc/product.d/*.prod, otherwise false.
+func moduleEnabledInProductFiles(identifier string) bool {
+	files, err := ioutil.ReadDir("/etc/products.d")
+	if err != nil {
+		return false
+	}
+	for _, file := range files {
```

Comment:
Are all files within that directory `.prod` files? Or is there anything else we need to skip inside?

Reply:
As far as I can see in the SUSEConnect and Zypper code, we can find the `baseprodict` which is a link for the `.prod` of the base product.

Reply:
Okay, one more thing: Can we only allow `.prod` files there? I have the feeling that we can break things up then some puts something else within that folder...

Reply:
Yeah, my idea in the first commit was consider `.prod` only. But now, I noticed that it did not do that. I'll change the code again to really only consider `.prod` files.

Reply:
Done